### PR TITLE
refac: Segregate infrastructure settings from calculation (business) args

### PR DESCRIPTION
**Pull request by BjarkeMeier**
Dependency injection has been implemented to inject infrastructure settings rather than having to pass them around the system.

### CODE REVIEW

File: source/databricks/calculation_engine/tests/infrastructure/test_get_calculator_args.py [hunk 1]
```diff
@@ -141,10 +141,10 @@ def test_parses_optional_time_series_points_table_name(
             with patch.dict("os.environ", job_environment_variables):
                 command_line_args = parse_command_line_arguments()
                 # Act
-                actual = create_calculation_arguments(command_line_args)
+                _, actual_settings = parse_job_arguments(command_line_args)
```
Comment (energinet-ajw) [inline source/databricks/calculation_engine/tests/infrastructure/test_get_calculator_args.py:144]: What does _ do?

File: source/databricks/calculation_engine/tests/calculator_job/test_start.py [hunk 2]
```diff
@@ -37,13 +36,13 @@ def test_exits_with_code_2(self):
 
 
 class TestWhenInvokedWithValidArguments:
-    def test_does_not_raise(self, any_calculator_args):
+    def test_does_not_raise(self, any_calculator_args, infrastructure_settings):
```
Comment (energinet-ajw) [inline source/databricks/calculation_engine/tests/calculator_job/test_start.py:39]: Is the plan to have different words: args and settings?

**Review by energinet-ajw — APPROVED** [review]

File: source/databricks/calculation_engine/tests/infrastructure/test_get_calculator_args.py (hunk 1 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/tests/infrastructure/test_get_calculator_args.py:144]: See no 2 [here](https://www.datacamp.com/tutorial/role-underscore-python).

File: source/databricks/calculation_engine/tests/calculator_job/test_start.py (hunk 2 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/tests/calculator_job/test_start.py:39]: The wheel is invoked with some arguments, but it reads some settings (from e.g. env variables).

**Pull request merged**